network-api/networkapi/management/commands/fix_alias_page_first_last_published.py

```diff
@@ -0,0 +1,26 @@
+from django.core.management.base import BaseCommand
+from django.db.models import Q
+from wagtail.core.models import Page
+
+
+class Command(BaseCommand):
+
+    def handle(self, **options):
+        aliases_without_first_last_published = (
+            Page.objects
+            .filter(alias_of__isnull=False)
+            .filter(Q(alias_of__first_published_at__isnull=False) | Q(alias_of__last_published_at__isnull=False))
+            .select_related('alias_of')
+        )
+        found = aliases_without_first_last_published.count()
+        if not found:
+            return self.stdout.write(self.style.WARNING("Found no alias pages to update"))
+
+        self.stdout.write(f"Found {found} alias page(s) to update")
+        for alias in aliases_without_first_last_published:
+            alias.first_published_at = alias.alias_of.first_published_at
+            alias.last_published_at = alias.alias_of.last_published_at
+            alias.save()
```

Comment:
since we're doing with a published alias, does this need a save_revision, too?

Reply:
Alias pages don't have revisions. The first revision gets created when the alias is converted to a full page - https://github.com/wagtail/wagtail/blob/716bf92c2dc2da2aca5e8f5aa6768b5b087cd4b0/wagtail/core/actions/convert_alias.py#L46